### DISCUSSION TITLE
Localhost not updating [Solved]

### DISCUSSION THREAD

**Discussion by 87prashant** [supabase/supabase#7286] [Questions] [answered]
I am new to Open-Source. My development on the local system is finished successfully, but whenever I do some changes in the code they are not reflected in the localhost. To see the changes I have to quit the local server and again do `npm run dev`
How to fix it any help?

**Comment by nattstack** [comment]
What did you use to build your project? For example `create-react-app`, when you save in your code editor it has a feature called hot reloading and shows live on your `localhost` website.

**Reply by 87prashant** [reply]
Hi @nattwasm,
I just forked and cloned the Supabase repo and then followed these commands according to the development.md 
`git clone https://github.com/<github_username>/supabase.git`
`cd supabase`
`cd apps/www`
`npm install`
`npm run dev`

Everything was successfully finished and the project is running on `localhost:3000` but not reflecting the changes as already mentioned.
I am totally new and don't know what I am missing

**Reply by 87prashant** [reply]
@nattwasm output of npm run dev:
if you can help?

![Screenshot 2022-06-21 233203](https://user-images.githubusercontent.com/106697681/174871652-b3022213-d17d-4fce-a393-73d3c2f2bb93.png)

**Reply by MahmoudAlaisami** [reply]
@87prashant this is the normal output from npm run dev, just go to the browser and type into the bar :  http://localhost:3000/

**Comment by 87prashant** [comment, marked as answer]
I am using WSL, with this we need to add `WATCHPACK_POLLING=true` in .env of the root of the project (for more info: [here](https://github.com/facebook/create-react-app/issues/10253))
